Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1268,6 +1268,16 @@ typedef struct {
 
 #endif // HAVE(UI_ASYNC_TEXT_INTERACTION)
 
+#if !defined(UI_SHIFT_KEY_STATE_ENUM)
+
+typedef NS_ENUM(NSInteger, UIShiftKeyState) {
+    UIShiftKeyStateNone = 0,
+    UIShiftKeyStateShifted,
+    UIShiftKeyStateCapsLocked
+};
+
+#endif
+
```

Comment:
Is this temporary??

Reply:
The `#ifdef` is for staging, since older versions of `UIAsyncTextInput.h` won't have these declarations.

Thanks for the review!